Revisions to fileUpload component

alderete-sfdc opened the pull request:
Edit and add comments. Refactoring. Minor functional changes. New bugs (probably).

Taost commented on force-app/main/default/lwc/fileUpload/fileUpload.js:
```diff
@@ -80,22 +85,30 @@ export default class FileUpload extends LightningElement {
+        await this.createContentDocumentLink(this.recordId, contentDocumentId);
       }
+
+      // Status and state updates
+      console.log(
+        "ContentDocument and ContentDocumentVersion records created."
+      );
```
nit, this log may be moved to the line 96 where the content and version records are created.

alderete-sfdc commented:
@Taost I'm #finally back from PTO, and would love to push this to completion this week. I've moved the logging line you commented on, and added a few additional log and comment lines. Otherwise unchanged.

Any idea why I'm failing the prettier and run-tests checks? Prettier just "exits with error code 1", and the test failure looks like it's elsewhere in the repo. I'm not sure how to resolve either of those...

Taost commented:
> @Taost I'm #finally back from PTO, and would love to push this to completion this week. I've moved the logging line you commented on, and added a few additional log and comment lines. Otherwise unchanged.
> 
> Any idea why I'm failing the prettier and run-tests checks? Prettier just "exits with error code 1", and the test failure looks like it's elsewhere in the repo. I'm not sure how to resolve either of those...

@alderete-sfdc Thanks for addressing the comment.

To fix the test, you can update the [test file here](https://github.com/salesforce/offline-app-developer-starter-kit/blob/main/force-app/main/default/lwc/fileUpload/__tests__/fileUpload.test.js#L24). The label has been changed from "Pick file to upload" to "Select file to upload".

To fix the prettier issue, you can run `npm run prettier` command in the Terminal window.

alderete-sfdc commented:
@Taost thanks for the tips! I was able to resolve the test failure (doh! who looks at tests! ;- ), and running prettier does make a change to one of the relevant-to-this-PR component files. I've committed those changes.

However, I am still getting prettier errors, on other components, and checking in the prettier changes for other components didn't resolve that. It also ran into conflicts with two of those components. I reverted that commit.

I'm reluctant to go mucking around in other people's code, because I'm not very involved with this repo (yet?), and don't know what considerations there are. 

Is there any way to turn this over to a real engineer who knows this repo well enough to resolve the issues safely?

dbreese commented:
@alderete-sfdc I just pulled your code down and ran `npm run prettier` and it just updates fileUpload.js with 2 minor updates. What errors are you seeing while running prettier?

Here's the file difference as shown in Fork for me showing the two added commas:

<img width="783" alt="image" src="https://github.com/salesforce/offline-app-developer-starter-kit/assets/2682021/59632e36-9c31-4c68-a5eb-9aad373eac1f">
